packages/components/src/components/FileUploader/FileUploader.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import type { ComponentMeta, ComponentStory } from "@storybook/react";
 import React, { useState } from "react";
-import Uppy from "@uppy/core";
+import UppyCore from "@uppy/core";
```

Comment:
Would using the explicit import fix it?
```suggestion
import { Uppy } from "@uppy/core";
```

Reply:
way better! Good call!